koji/vars/external_repos.yml

```diff
@@ -8,40 +8,20 @@ external_repos:
   - name: epel-6
     url: file:///mnt/koji/external-repos/www/epel6-$arch/RPMS.epel/
   # EL7 repositories
-  - name: rhel-7.0-server
```

Comment:
This is used in https://github.com/theforeman/tool_belt/blob/master/configs/pulp/3.nightly.yaml but I think that isn't really used anymore since we have pulpcore, right?

Reply:
Ahh yea, it does appear some Pulp 3 nightly tags use this external repo. @evgeni can you confirm if that tag setup is actively used? And should we update it to CentOS?

Reply:
Ok so those Pulp 3 "nightly" tags were quite old and so I have opened to remove that config https://github.com/theforeman/tool_belt/pull/374 then we can drop this repo